mmdet/apis/ote/extension/datasets/data_utils.py

```diff
@@ -295,3 +296,45 @@ def create_gt_box(x1, y1, x2, y2, label_name):
         dataset_items.append(dataset_item)
 
     return dataset_items
+
+
+def get_sizes_from_DatasetEntity(dataset: DatasetEntity, target_wh: list):
```

Comment:
```suggestion
def get_sizes_from_dataset_entity(dataset: DatasetEntity, target_wh: list):
```
And don't forget to change it everywhere it is used.

Reply:
Thank you